src/Chapter1.hs

```diff
@@ -520,7 +521,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if (abs x) < (abs y) then x else y
```

Comment:
Here as well 🙂 

```suggestion
closestToZero x y = if abs x < abs y then x else y
```

Reply:
I setup the Vscode Haskell plugin which seems to act up at times, but has begun to reliably show me redundant parens :)